spec/redux-test.js

```diff
@@ -0,0 +1,31 @@
+import * as actions from '../src/redux/actionCreators';
+import * as types from '../src/redux/action-types';
+import running from '../src/redux/reducer';
+
+describe('The action creator', function() {
+  it('run should return return a object w/action.payload === true', function() {
+    var expectedAction = {
+      type: types.RUNNING,
+      payload: true
+    };
+    expect(actions.run()).toEqual(expectedAction);
+  });
+  it('run should return return a object w/action.payload === false', function() {
+    var expectedAction = {
+      type: types.RUNNING,
+      payload: false
+    };
+    expect(actions.stop()).toEqual(expectedAction);
+  });
+});
+
+
+describe('The reducer', function() {
+  it('should return the initial state', function() {
+    expect(running(undefined, {})).toEqual({running: false});
+  });
+  it('should handle RUNNING', function() {
+    expect(running(undefined, actions.run())).toEqual({running: true});
+    expect(running({running: true}, actions.stop())).toEqual({running: false});
```

Comment:
by using `toEqual()` with the entire state object, you are actually making the test somewhat brittle: i.e. a small code change that doesn't actually break anything will break this test, leading to a false negative. A better way to write this would be:

```javascript
expect(running(undefined, actions.run()).running).toBeTrue();
```